doc-Monitoring_Alerts_and_Reporting/topics/Operational_Alerts.adoc

```diff
@@ -158,7 +158,7 @@
 					
 | 
 						
-							Something is filling the disk such as temp files used by the operating system such as, yum updates and normal /tmp files, or EVM temp files in /var/lib/data/miqtemp/.
+							Something is filling the disk such as temp files used by the operating system such as, yum updates and normal /tmp files, or temp files in /var/lib/data/miqtemp/.
```

Comment:
Very small thing, but the comma after the second "such as" should be removed -- or maybe could the sentence be reworded so there are not two "such as" instances in it?